source/Google/Maps/ApiDefinition.cs

```diff
@@ -562,6 +600,34 @@ interface MapViewDelegate
 		// - (void)mapViewSnapshotReady:(GMSMapView *)mapView;
 		[Export ("mapViewSnapshotReady:"), EventArgs ("GMSSnapshotReady")]
 		void SnapshotReady (MapView mapView);
+
+		// @optional -(void)mapView:(GMSMapView * _Nonnull)mapView didChangeMapCapabilities:(GMSMapCapabilityFlags)mapCapabilities;
+		[Export ("mapView:didChangeMapCapabilities:"), EventArgs ("GMSMapCapabilities"), EventName ("MapCapabilitiesChanged")]
+		void DidChangeMapCapabilities (MapView mapView, MapCapabilityFlags mapCapabilities);
+
+		// @optional -(void)mapView:(GMSMapView * _Nonnull)mapView didTapFeatures:(NSArray<id<GMSFeature>> * _Nonnull)features inFeatureLayer:(GMSFeatureLayer * _Nonnull)featureLayer atLocation:(CLLocationCoordinate2D)location;
+		[Export ("mapView:didTapFeatures:inFeatureLayer:atLocation:"), EventArgs ("GMSFeaturesTapped"), EventName ("FeaturesTapped")]
+		void DidTapFeatures (MapView mapView, Feature [] features, FeatureLayer featureLayer, CLLocationCoordinate2D location);
+	}
+
+	// @interface GMSMapViewOptions : NSObject
+	[BaseType (typeof (NSObject), Name = "GMSMapViewOptions")]
+	interface MapViewOptions {
+		// @property (nonatomic) CGRect frame;
+		[Export ("frame", ArgumentSemantic.Assign)]
+		CGRect Frame { get; set; }
+
+		// @property (nonatomic) GMSCameraPosition * _Nullable camera;
+		[NullAllowed, Export ("camera", ArgumentSemantic.Assign)]
+		CameraPosition Camera { get; set; }
+
+		// @property (nonatomic) GMSMapID * _Nullable mapID;
+		[NullAllowed, Export ("mapID", ArgumentSemantic.Assign)]
+		MapID MapID { get; set; }
```

Comment:
Please rename to `MapId MapId` do be in line with standard .NET naming conventions